.circleci/config.yml

```diff
@@ -205,9 +205,20 @@ workflows:
           docker_tag: dev-snapshot
           requires:
             - test_cli_staging
+      - codacy/sbt:
+          name: publish_cli_jar
+          install_sbt_version: 1.3.12
+          cmd: |
+            sbt "set codacyAnalysisCli / version := \"$(cat .version)\";
+                 set test in assembly := {};
+                 codacyAnalysisCli/assembly"
+            mkdir -p ~/workdir/artifacts/
+            mv ./cli/target/scala-2.12/codacy-analysis-cli-assembly-$(cat .version).jar ~/workdir/artifacts/
+          requires:
+            - test_cli_staging
       - codacy/publish_ghr:
           context: CodacyGitHub
-          path: ~/
+          path: ~/workdir/artifacts/
```

Comment:
What is the idea here? I don't know this codacy/publish_ghr, what does it do?

Reply:
So, basically we are generating the fat jar and creating a release folder with that fat jar in `~/workdir/artifacts/`.
This `codacy/publish_ghr` creates a GitHub release with all the artifacts in the specified folder. In this case, it will contain the jar created on the previous step.

Reply:
Got it! Fun stuff.